test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -198,6 +195,14 @@ def test_single_mom_hook_failure_affects_vnode(self):
                            interval=2)
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid2)
 
+        # Restore original node setup for future test cases.
+        self.server.cleanup_jobs(extend='force')
+        self.server.manager(MGR_CMD_DELETE, NODE, id="@default",
+                            expect=True)
+        for m in self.moms.values():
+            self.server.manager(MGR_CMD_CREATE, NODE,
+                                id=m.shortname)

```

Comment:
You shouldn't have to do this.  revert_to_defaults() should be doing this.  Does the test fail if you remove it?

Reply:
Yes

Reply:
@hirenvadalia any chance you could take a peek at the code I'm commenting on?  It's at the end of the last few tests where @bayucan has to delete all the nodes and recreate the natural vnode.  I thought this was all done in revert_to_defaults().

Reply:
@bhroam No revert_to_defaults() don't delete nodes and re-create it (except on Cray), revert_to_defaults() only reverts attributes of natural vnode and it will delete all vnodes and vnode definition files.